crates/egui_glow/src/winit.rs

```diff
@@ -53,14 +54,15 @@ impl EguiGlow {
             repaint_after,
             textures_delta,
             shapes,
-        } = self.egui_ctx.run(raw_input, run_ui);
+            ..
+        } = self.egui_ctx.run(raw_input, ViewportIdPair::MAIN, run_ui);
 
         self.egui_winit
             .handle_platform_output(window, &self.egui_ctx, platform_output);
 
         self.shapes = shapes;
         self.textures_delta.append(textures_delta);
-        repaint_after
+        repaint_after.last().map(|(_, t)| *t).unwrap_or_default()
```

Comment:
why only look at the last here?

Reply:
This is because glow implementation do not have support for multiples viewports!

Glow implementation should have support for multiples windows?